frontend/spec/features/resources_spec.rb

```diff
@@ -425,6 +425,20 @@
     expect(elements.length).to eq 1
   end
 
+  it 'has the Include URIs checkbox checked by default on the EAD Export dropdown menu' do
+    now = Time.now.to_i
+    resource = create(:resource, title: "Resource Title #{now}")
+    run_index_round
+
+    visit "resources/#{resource.id}"
+
+    element = find('#form_download_ead', visible: false)
+    expect(element).to have_css '#include-uris', visible: false
+
```

Comment:
@thimios Lines 435-436 above aren't really doing anything except repeating line 438 below, and the main test lies in line 439, so seems like 435/6 can just be deleted?